Code/Mantid/docs/source/algorithms/CalMuonDetectorPhases-v1.rst

```diff
@@ -0,0 +1,65 @@
+
+.. algorithm::
+
+.. summary::
+
+.. alias::
+
+.. properties::
+
+Description
+-----------
+
+Calculates detector asymmetries and phases from a reference dataset. The algorithm fits each of
+the spectra in the input workspace to:
+
+.. math:: f_i(t) = A_i \sin\left(\omega t + \phi_i\right)
+
+where :math:`\omega` is shared across spectra and :math:`A_i` and :math:`\phi_i` are
+detector-dependent. The algorithm outputs a table workspace containing the detector ID (i.e. the
+spectrum index), the asymmetry and the phase. This table is intended to be used as the input
+*PhaseTable* to :ref:`PhaseQuad <algm-PhaseQuad>`. In addition, the fitting results are returned
+in a workspace group, where each of the items stores the original data (after removing the
+exponential decay), the data simulated with the fitting function and the difference between data
+and fit as spectra 0, 1 and 2 respectively.
+
+There are three optional input properties: *FirstGoodData* and *LastGoodData* define the fitting range.
+When left blank, *FirstGoodData* is set to the value stored in the input workspace and *LastGoodData*
```

Comment:
Maybe this could explain more explicitly that "stored in the input workspace..." ... in the log entry FirstGoodData?